services/account_type/cloudtrail_owner.yaml

```diff
@@ -36,18 +36,12 @@ Resources:
         - Key: cloudzero-reactor-account-id
           Value: !Ref ReactorAccountId
 
-  SnsTopic:
```

Comment:
This was a bug. We don't want to create a _new_ topic; we want to use the one passed in the arguments.